Tests/BankAccount/Business/HyperwalletBankAccountBusinessTests.swift

```diff
@@ -0,0 +1,52 @@
+import Hippolyte
+@testable import HyperwalletSDK
+import XCTest
+
+// swiftlint:disable force_cast
+class HyperwalletBankAccountBusinessTests: XCTestCase {
+    override func setUp() {
+        Hyperwallet.setup(HyperwalletTestHelper.authenticationProvider)
+    }
+
+    override func tearDown() {
+        if Hippolyte.shared.isStarted {
+            Hippolyte.shared.stop()
+        }
+    }
+
+    func testCreateBankAccount_business_success() {
+        // Given
+        let expectation = self.expectation(description: "Create bank account completed")
+        let response = HyperwalletTestHelper.okHTTPResponse(for: "BankAccountBusinessResponse")
+        let url = String(format: "%@/bank-accounts", HyperwalletTestHelper.userRestURL)
+        let request = HyperwalletTestHelper.buildPostResquest(baseUrl: url, response)
+        HyperwalletTestHelper.setUpMockServer(request: request)
+
+        var bankAccountResponse: HyperwalletBankAccount?
+        var errorResponse: HyperwalletErrorType?
+
+        // When
+        let bankAccount = HyperwalletBankAccount
+            .Builder(transferMethodCountry: "US",
+                     transferMethodCurrency: "USD",
+                     transferMethodProfileType: "BUSINESS")
+            .bankAccountId("7861012345")
+            .branchId("102000021")
+            .bankAccountRelationship(.ownCompany)
+            .bankAccountPurpose(.checking)
+            .build()
```

Comment:
set profile fields as well for Business and Individual Tests

Reply:
Done